support-workers/src/main/scala/com/gu/emailservices/PaperEmailFields.scala

```diff
@@ -0,0 +1,69 @@
+package com.gu.emailservices
+
+import com.gu.i18n.Currency
+import com.gu.salesforce.Salesforce.SfContactId
+import com.gu.support.catalog.{FulfilmentOptions, HomeDelivery, ProductOptions}
+import com.gu.support.workers._
+import org.joda.time.LocalDate
+
+case class PaperEmailFields(
+  subscriptionNumber: String,
+  fulfilmentOptions: FulfilmentOptions,
+  productOptions: ProductOptions,
+  billingPeriod: BillingPeriod,
+  user: User,
+  paymentSchedule: PaymentSchedule,
+  firstDeliveryDate: Option[LocalDate],
+  currency: Currency,
+  paymentMethod: PaymentMethod,
+  sfContactId: SfContactId,
+  directDebitMandateId: Option[String] = None
+) extends EmailFields {
+
+  val dataExtension = fulfilmentOptions match {
+    case HomeDelivery => "paper-delivery"
+    case _ => "paper-voucher"
+  }
+
+  val firstPaymentDate = SubscriptionEmailFieldHelpers.firstPayment(paymentSchedule).date
+
+  val paymentFields = paymentMethod match {
+    case dd: DirectDebitPaymentMethod => List(
+      "bank_account_no" -> mask(dd.bankTransferAccountNumber),
+      "bank_sort_code" -> hyphenate(dd.bankCode),
+      "account_holder" -> dd.bankTransferAccountName,
+      "payment_method" -> "Direct Debit",
+      "mandate_id" -> directDebitMandateId.getOrElse("")
+    )
+    case _: CreditCardReferenceTransaction => List("payment_method" -> "Credit/Debit Card")
+    case _: PayPalReferenceTransaction => List("payment_method" -> "PayPal")
+  }
+
+  val deliveryAddressFields = user.deliveryAddress.map { address =>
+    List(
+      "delivery_address_line_1" -> address.lineOne.getOrElse(""),
+      "delivery_address_line_2" -> address.lineTwo.getOrElse(""),
+      "delivery_address_town" -> address.city.getOrElse(""),
+      "delivery_postcode" -> address.postCode.getOrElse(""),
+      "delivery_country" -> address.country.name
+    )
+  }
+
+  override val fields = List(
```

Comment:
Based on: https://github.com/guardian/subscriptions-frontend/blob/9ae6ee3d9f8d5f21bce81b2c76225d7fc07db6fd/app/model/Email.scala#L143-L207

Reply:
However there are a few omissions:

- "IncludesDigipack" - I wrote a little helper function for this initially but haven't included it because this value doesn't seem to be used anyway!
- "delivery_county" - we don't collect this anymore
- "delivery_instructions" - we don't collect these as part of the new flow. Not sure if that's intentional @SarahLaughton?
- "title" - this can be wired through if necessary / if we want the emails to look more formal